tlscanary/js/scan_worker.js

```diff
@@ -123,7 +123,11 @@ function collect_request_info(xhr, report_certs) {
         sec_info.QueryInterface(Ci.nsITransportSecurityInfo);
         info.transport_security_info_status = true;
         info.security_state = sec_info.securityState;
-        info.raw_error = sec_info.errorMessage;
+        try {
+            info.raw_error = sec_info.errorMessage;
```

Comment:
This shouldn't throw. The `sec_info` object has no `errorMessage` attribute but should be `undefined` (yay javascript).